frontend/src/components/footer/styles.scss

```diff
@@ -33,3 +33,12 @@
 .socials {
   gap: 0.75rem;
 }
+
+.react-tooltip#shareProject {
+  border: 1px solid #2c3038;
+}
+
+.react-tooltip__place-top#shareProject > .react-tooltip-arrow {
+  border-bottom: 1px solid #2c3038;
+  border-right: 1px solid #2c3038;
+}
```

Comment:
I'll take a look at this.

Reply:
This (and the `#dueDateBoxTip`) issue was caused by me fixing an issue from a dependency update, and then later going back and standardizing all the `Tooltip` ids to have `Tooltip` in them (specifically to avoid an accidental collision). I've fixed both locally, and I'll do a push once I've fixed some other issues.